<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgChapter</class>
 <widget class="QDialog" name="DlgChapter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chapter properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="11" column="0" colspan="2">
    <layout class="QHBoxLayout" name="BottomLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKBT">
       <property name="toolTip">
        <string>Accept modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset resource="../RSCffDiaporama.qrc">
         <normaloff>:/img/action_ok.png</normaloff>:/img/action_ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBt">
       <property name="toolTip">
        <string>Cancel modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../RSCffDiaporama.qrc">
         <normaloff>:/img/action_cancel.png</normaloff>:/img/action_cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HelpBt">
       <property name="toolTip">
        <string>Show help about this dialogue box</string>
       </property>
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset resource="../RSCffDiaporama.qrc">
         <normaloff>:/img/help.png</normaloff>:/img/help.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ChapterNameLabel">
     <property name="text">
      <string>Chapter name:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ChapterEventDateLabel">
     <property name="text">
      <string>Event date:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="OverrideProjectDateCB">
     <property name="toolTip">
      <string>If this box is checked, the date of the chapter is different from the date of the project.

Remark: It is often useful to differentiate the date of the chapter of the date of the project.
For example, if the project is &quot;My vacation from 1 to 15&quot;, the chapter may be limited to the 3.</string>
     </property>
     <property name="text">
      <string>Chapter date different from project date</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>164</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="ChapterEventDateED">
     <property name="toolTip">
      <string>Select a date for the chapter.</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="StartNewChapterCB">
     <property name="toolTip">
      <string>Start a new chapter from this slide. (If this box is checked, the slide will mark the beginning of a new chapter.)
Remarks : This box cannot be deactivated for the first slide of the project.</string>
     </property>
     <property name="text">
      <string>Start a new chapter with this slide</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="LocationLabel">
     <property name="text">
      <string>Event location:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="OverrideProjectLocationCB">
     <property name="toolTip">
      <string>If this box is checked, the date of the chapter is different from the date of the project.

Remark: It is often useful to differentiate the date of the chapter of the date of the project.
For example, if the project is &quot;My vacation from 1 to 15&quot;, the chapter may be limited to the 3.</string>
     </property>
     <property name="text">
      <string>Chapter location different from project location</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="LocationFrame">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="LocationIcon">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame {
background-color: transparent;
border: 0;
}</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Location">
          <property name="styleSheet">
           <string notr="true">QFrame {
background-color: transparent;
border: 0;
}</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="LocationBT">
       <property name="toolTip">
        <string>Select the location of the event of the project</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../RSCffDiaporama.qrc">
         <normaloff>:/img/EditGMaps.png</normaloff>:/img/EditGMaps.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ClearLocationBT">
       <property name="toolTip">
        <string>Clear this location</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../RSCffDiaporama.qrc">
         <normaloff>:/img/trash.png</normaloff>:/img/trash.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="ChapterNameED">
     <property name="toolTip">
      <string>Enter here the name of the chapter</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="OverrideDateLabel">
       <property name="text">
        <string>Override date:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="OverrideDateCB">
         <property name="toolTip">
          <string>If this box is checked, you can enter a longtime different from that generated from the date of the chapter.</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="ChapterDateED">
       <property name="toolTip">
        <string>Enter here a date, in a free format, for the chapter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>StartNewChapterCB</tabstop>
  <tabstop>ChapterNameED</tabstop>
  <tabstop>OverrideProjectDateCB</tabstop>
  <tabstop>ChapterEventDateED</tabstop>
  <tabstop>ChapterDateED</tabstop>
  <tabstop>OverrideProjectLocationCB</tabstop>
  <tabstop>LocationBT</tabstop>
  <tabstop>ClearLocationBT</tabstop>
  <tabstop>OKBT</tabstop>
  <tabstop>CancelBt</tabstop>
  <tabstop>HelpBt</tabstop>
 </tabstops>
 <resources>
  <include location="../RSCffDiaporama.qrc"/>
 </resources>
 <connections/>
</ui>
